aspnet/includes/identity/identity_replace_membership.md

```diff
@@ -0,0 +1,8 @@
+> [!NOTE]
+> Since this article was written, the ASP.NET Membership providers have been superseded by [ASP.NET Identity](/aspnet/identity/overview/getting-started/introduction-to-aspnet-identity). We strongly recommend updating apps to use the [ASP.NET Identity](/aspnet/identity/overview/getting-started/introduction-to-aspnet-identity) platform rather than the Membership providers featured at the time this article was written. ASP.NET Identity has a number of advantages over the ASP.NET Membership system, including :
+>
+> * More secure.
+> * More preformant.
+> * Improved testability.
+> * Support for two-factor authentication.
+> * More features.
```

Comment:
"More features.' feels unneccessary, and "More secure" is a bit amorphous. Instead, I might include "Claims-based Identity" and "Greater interoperability with ASP.Net Core".

```suggestion
> * Better performance
> * Improved extensibility and testability
> * Support for OAuth, OpenID Connect, and two-factor authentication
> * Claims-based Identity support
> * Better interoperability with ASP.Net Core
```

Reply:
@StephenMolloy I took your suggestion, however
>  "More secure" is a bit amorphous. 

Of the bullets listed, 99.99% won't care as their legacy apps won't add OAuth, don't care about perf, etc. The only thing they'd care about is ***more secure***. Last chance to let me add ***more secure*** back.